src/main/webapp/script.js

```diff
@@ -135,17 +137,22 @@ function viewRoute(route) {
     });
   }
 
+  let ratingScore = generateRating(route);
   let commentsPanel = createCommentsPanel(route);
+  let commentForm = createCommentForm(route);
 
   // Add button to exit preview mode.
   let backButton = document.createElement("button");
   backButton.innerHTML = "BACK TO ROUTE CREATION";
+  backButton.style.background = "#ffb3b3";
```

Comment:
Prefer setting a class rather than a style property directly. If all styles are controlled by CSS, you can change the look and feel in a single place. If some of it is done in JS, you need to search for the location in multipe places.

Reply:
Done